schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -1007,6 +1007,19 @@
                 priority="100"/>
 
 
+  <!-- Use gmd:specification label for the specification title -->
+  <xsl:template mode="render-field"
+                match="gmd:specification/*/gmd:title"
+                priority="100">
+    <dl>
+      <dt><xsl:value-of select="tr:nodeLabel(tr:create($schema, @code), 'gmd:specification', 'gmd:CI_Citation')"/></dt>
+      <dd>
+        <xsl:apply-templates mode="render-value" select="."/>
+      </dd>
+    </dl>
+  </xsl:template>
+
+
```

Comment:
This is not necessary as you avoid above only `gmd:identificationInfo/*/gmd:citation/*/gmd:title` and other fields should be rendered as usual.

```suggestion
```

Reply:
The problem is for the specification title field, displays a generic `Title` label. I wanted to use a more clear label `Specification`.

Reply:
If you set the label with a context in `labels.xml` it should also work and we have same label in view and edit mode?

Reply:
ok, committing your suggestion and will manage in the `labels.xml` in other pull request.